projects/fsastorefrontlib/src/assets/translations/en/dashboard.en.ts

```diff
@@ -0,0 +1,14 @@
+export const dashboard = {
+  dashboard: {
+    name: 'Name',
+    email: 'Email',
+    noCustomers: '[DE] You haven\'t assigned any customers yet', // prettier-ignore
```

Comment:
Should be without [DE]

Reply:
Also to avoid use of `// prettier-ignore` you can use template literals or backtick instead of simple quotes ''.

Reply:
> 

Yes, but majority of devs have double quotes transforming to single quotes on save. So putting this inside would potentially cause future problems for somebody working on this file. Backticks are a good solution, but I choose this because of consistency in code, since Spartacus isn't using them anywhere.